src/main/java/uk/gov/hmcts/ccd/domain/service/getevents/AuthorisedGetEventsOperation.java

```diff
@@ -61,6 +66,13 @@ public AuthorisedGetEventsOperation(@Qualifier("classified") GetEventsOperation
         return secureEvents(caseTypeId, event.getCaseDataId(), singletonList(event)).stream().findFirst();
     }
 
+    private List<AuditEvent> secureEvents(List<AuditEvent> events) {
+        if (null == events || events.size() == 0) {
+            return Lists.newArrayList();
+        }
```

Comment:
we sure this is needed? DefaultGetEventsOperation always returns at least 1 event, the create event for a case.
And I think the other decorators that execute before this one will already return an empty list in case that single event is not accessible

Reply:
removed above check